ui5.yaml

```diff
@@ -1,21 +1,27 @@
-specVersion: "2.0"
+specVersion: "3.0"
 metadata:
   name: ui5.typescript.helloworld
 type: application
 framework:
   name: OpenUI5
-  version: "1.104.0"
+  version: "1.112.0"
   libraries:
     - name: sap.m
     - name: sap.ui.core
     - name: sap.ui.unified
-    - name: themelib_sap_fiori_3
+    - name: themelib_sap_horizon
+builder:
+  customTasks:
+    - name: ui5-tooling-transpile-task
+      afterTask: replaceVersion
+      configuration: &cfgTranspile
+        transpileTypeScript: true
+        removeConsoleStatements: true
 server:
   customMiddleware:
-  - name: ui5-middleware-livereload
-    afterMiddleware: compression
-    configuration:
-      debug: true
-      extraExts: "xml,json,properties"
-      port: 35729
-      path: "webapp"
+    - name: ui5-tooling-transpile-middleware
+      afterMiddleware: compression
+      configuration:
+        <<: *cfgTranspile
+    - name: ui5-middleware-livereload
+      afterMiddleware: compression
```

Comment:
Somehow using the UI5 tools only ADDS complexity.
Here some complex configuration to be added which normal people will hardly write easily without looking it up each time. 
.babelrc still needed, even longer. Only package.json gets shortened, but the stuff that is removed is mostly calls to Babel, which are "standard". Seems like we are just replacing _standard_ tools with _proprietary_ tools here. Not sure this is so much better, in particular for teaching how the setup works.
I think I'll add the previous setup to a "back to basics" branch of its own.

Reply:
The `.babelrc` isn't needed at all! I just showcase it how to use it to not completely rewrite the hello world sample